doc/man1/flux-mini.rst

```diff
@@ -174,6 +174,10 @@ following additional job parameters:
    option is minutes when no units are otherwise specified. If the time
    limit is unspecified, the job is subject to the system default time limit.
 
+**-f, --flags=FLAGS**
+   Set job submission flags.  The flag ``waitable`` will allow the job to
```

Comment:
We might as well document `debug` and `novalidate` flags. I don't think they will go away or change, and `novalidate` at least might be useful for users wanting to avoid validation in their batch jobs.